app/src/main/java/org/dicekeys/app/fragments/ApiRequestFragment.kt

```diff
@@ -177,8 +175,10 @@ class ApiRequestFragment : AbstractListDiceKeysFragment<ApiRequestFragmentBindin
             urlCommand.executeCommand()
 
             if(urlCommand.hasException()){
-                urlCommand.sendException()
-                findNavController().popBackStack()
+                dialog(getString(R.string.error), urlCommand.exception?.message ?: "Something went wrong") {
```

Comment:
How about:
   "Another app made an invalid request of the DiceKeys app:" + (urlCommand.exception?.message ?: "unknown error")

Reply:
Is this correct?
"Another app made an invalid request **of** the DiceKeys app: %s"

Reply:
I'm forgetting my Kotlin, but looks right